test/integration/display-test.js

```diff
@@ -1,11 +1,11 @@
 "use strict";
 
-const chai = require("chai"),
-  chaiHttp = require("chai-http"),
+const fs = require('fs'),
+  expect = require('chai').expect,
+  request = require("superagent"),
```

Comment:
@stulees chaiHttp does not accept setting certificates. So I had to change to superagent. cheers